src/almanack/checks/data.py

```diff
@@ -6,12 +6,61 @@
 import shutil
 import tempfile
 from datetime import datetime, timezone
-from typing import Optional, Tuple
+from typing import Any, Dict, Optional, Tuple
 
 import pygit2
+import yaml
 
-from .calculate_entropy import calculate_aggregate_entropy, calculate_normalized_entropy
-from .git_operations import clone_repository, get_commits, get_edited_files
+from ..git import clone_repository, get_commits, get_edited_files
+from .entropy.calculate_entropy import (
+    calculate_aggregate_entropy,
+    calculate_normalized_entropy,
+)
+
+CHECKS_TABLE = f"{pathlib.Path(__file__).parent!s}/checks.yml"
+
+
+def get_table(repo_path: str) -> Dict[str, Any]:
+    """
+    Perform checks on a repository and return the results in a structured format.
```

Comment:
again, what does it mean to "check" a repository? I think a brief summary or a couple adjectives would help here

Reply:
Thanks again, I'll move to `metrics` here instead.